package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "ios": "expo start --ios",
     "android": "expo start --android",
-    "dev": "BABEL_ENV=web next",
+    "dev": "yarn concurrently \"BABEL_ENV=web next\" \" yarn server\" ",
```

Comment:
Can you pls create also command just for web?
Sometimes I would like to run these commands separately thanks.

Reply:
sure